verify/verify.go

```diff
@@ -925,41 +929,67 @@ func getMatchingTcbLevel(tcbLevels []pcs.TcbLevel, tdReport *pb.TDQuoteBody, pck
 	return pcs.TcbLevel{}, fmt.Errorf("no matching TCB level found")
 }
 
-func checkQeTcbStatus(tcbLevels []pcs.TcbLevel, isvsvn uint32) error {
+func readQeTcbStatus(tcbLevels []pcs.TcbLevel, isvsvn uint32) (pcs.TcbLevel, error) {
 	for _, tcbLevel := range tcbLevels {
```

Comment:
Just curious: This is not related to your changes but more about the original logic: So if we loop through 'tcbLevels' and return the first one that satisfies 'tcbLevel.Tcb.Isvsvn <= isvsvn'. Could we be missing something? For example, if `tcbLevels` are: {{1, outdated}, {2, uptodate}}, and `isvsvn`=2. We go through `tcbLevels` and found the first one meets the condition `tcbLevel.Tcb.Isvsn<=isvsvn` and return it, and in `checkQeTcbStatus` we found it's outdated. However it's actually up to date since the second uptodate entry meets the requirement.

Is this possible? It may be that I'm missing some info that Intel PCS won't return such results. If so, some comments here to clarify this will be helpful.

Reply:
This is unlikely to happen because Intel's published algorithm will go through the `tcbLevels` in an descending order
![image](https://github.com/user-attachments/assets/6552572e-1b34-4daf-aaaf-0fb209b6bae3).

Also based on my observation from Intel PCS response, the returned `tcbLevels` results are already sorted in a descending order.

Reply:
That makes sense. Can you add a comment here to explain this (Intel's results are in descending order)?

Reply:
Done, also added comments for TDX TCB info for clarity.